airflow/contrib/operators/dataproc_operator.py

```diff
@@ -377,6 +377,132 @@ def execute(self, context):
         self._wait_for_done(service)
 
 
+class DataprocClusterScaleOperator(BaseOperator):
+    """
+    Scale a cluster on Google Cloud Dataproc. The operator will wait until the
+    cluster is re-scaled.
+
```

Comment:
Can you just add a link to `https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/scaling-clusters` and say "For more information visit {LINK}".

Reply:
Link added.